Assignment/solution_student/css/style.css

```diff
@@ -6,18 +6,175 @@ Developer: @poopsplat in Chicago
 */
 
 /******************************************
+/* SETUP
+/*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+/******************************************
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
-  font-size: 16px;
-  background: white;
+  	color: #121212;
+  	font-size: 16px;
+  	font-weight: 300;
+  	margin-top: 0px;
+}
+
+h1{
+	font-family: 'Open Sans', sans-serif;
```

Comment:
Since I believe Open Sans is used everywhere on the page, you could eliminate having to repeat your `font-family` declaration on everything by applying it to the `body` so that everything on the page just naturally inherits it, make sense? Saves a lot of writing for you!